build.sh

```diff
@@ -57,6 +57,24 @@ do
   esac
 done
 
+if [ "$DO_UPDATE" == true ] && [ "$QUIET" == false ]; then
+  echo "Checking if html-build is up to date..."
+  ORIGIN_URL=$(git remote get-url origin)
+  git fetch -q $ORIGIN_URL master
```

Comment:
FETCH_HEAD is simply the place where the ref ends up if you specify both a remote/url and a branch, but no target branch as in `git fetch origin master:newbranch`. I'm not worried about stomping on it, if people have done a fetch like this and then not immediately used FETCH_HEAD, then they probably won't mind us overwriting it.

Even passing -v to git fetch doesn't show any interesting information I'm afraid, it's still like the above pasted.